AutoCollection/WebSnippet/fetchSnippet.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const fs = require('fs');
+const https = require('https');
+
+let snippetUrl = 'https://raw.githubusercontent.com/microsoft/ApplicationInsights-JS/master/AISKU/snippet/snippet.js';
```

Comment:
I thought you were creating an npm package to avoid this

Reply:
Yes, I will update the PR soon! Thank you Hector!